apps/docs/src/app/core/component-docs/shellbar/examples/shellbar-side-nav/shellbar-side-nav-example.component.html

```diff
@@ -9,32 +9,32 @@
     ></button>
 
     <fd-shellbar-logo>
-        <a href="#" class="fd-shellbar__logo fd-shellbar__logo--image-replaced" aria-label="SAP"></a>
+        <a href="#" class="fd-shellbar__logo fd-shellbar__logo--image-replaced" alt="SAP Logo"></a>
     </fd-shellbar-logo>
 </fd-shellbar>
 <div class="shellbar-sidenav-content-example">
-    <fd-side-nav [condensed]="condensed">
+    <fd-side-nav [condensed]="condensed" role="navigation" aria-roledescription="Side Navigation">
         <div fd-side-nav-main>
-            <ul fd-nested-list [textOnly]="false">
-                <li fd-nested-list-item>
+            <ul fd-nested-list [textOnly]="false" role="tree" aria-roledescription="Navigation List Tree">
+                <li fd-nested-list-item role="listitem">
```

Comment:
Same here, it can easily be handled internally and avoid setting it manually in every example.

Reply:
fixed